controllers/nnf_workflow_controller_test.go

```diff
@@ -167,6 +167,14 @@ var _ = Describe("NNF Workflow Unit Tests", func() {
 				FileSystemType: fsType,
 				AllocationSets: []nnfv1alpha1.NnfStorageAllocationSetSpec{},
 			},
+			Status: nnfv1alpha1.NnfStorageStatus{
```

Comment:
NnfStorage defines Status as a subresource, so the `k8sClient.Create()` below will not store this status section.